python/cugraph/tests/test_force_atlas2.py

```diff
@@ -120,7 +123,7 @@ def test_force_atlas2(graph_file, score, max_iter,
         iterations on a given graph.
     """
 
-    matrix_file = graph_file[:-4] + ".mtx"
+    matrix_file = PurePath(graph_file).with_suffix(".mtx")
```

Comment:
`graph_file` should already be a `PurePath` object, so no need to construct a new one, in other words, you should be able to just do:

`matrix_file = graph_file.with_suffix(".mtx")`

Reply:
right, there is no need to cast graph_file since it is already a `PurePath`